dbms/src/Functions/FunctionsRegexp.h

```diff
@@ -1698,7 +1700,7 @@ class FunctionStringRegexpSubstr : public FunctionStringRegexpBase
 
         auto null_map_col = ColumnUInt8::create();
         typename ColumnUInt8::Container & null_map = null_map_col->getData();
-        null_map.resize(col_size, 1);
+        null_map.resize(col_size);
```

Comment:
Why change this? If `null_map` is not initialized to all `1`s, L1715 should need to assign all the element in `null_map_col` to 1.